config/initializers/rack_attack.rb

```diff
@@ -0,0 +1,15 @@
+
+# Rack attack protection
+module Rack
+  class Attack
+    throttle('req/ip', limit: 120, period: 1.minute, &:ip)
+
+    blocklist('auth-less annoyances') do |req|
```

Comment:
True, I'll fix that.
I haven't imported `Tokenator` yet as it is in form of a middleware and it won't be a middleware on our side. I'll add it once I find the right place for it.